ls1intum/Artemis
# [Enhancement] - Multiple adaptions in the exam scores view

### Checklist

### Motivation and Context


We want to have some little adaptions to the exam scores page, since some aspects are not ideally

### Description


1. Removed some whitespaces from the exam information below the exam title (with number of points, exercise groups etc.)
2. Added percentage value of how many students passed (if grading key defined) and how many percentage actually submitted
3. Adapted the exercise average scores, made the bars smaller (less height), added a possibility to click on the bar to navigate into the scores page of the corresponding exercise
4. Added a new line in the exam information matrix which shows the standard grade deviation (only if grade scale defined and grades only consist of numeric values!)
5. Explicitly add points or grades to the matrix to distinguish between those

### Steps for Testing


1. Log in to Artemis
2. Navigate into the scores page of an exam (preferably with second correction done and grade key defined, in order to check everything)
3. Check the improvements mentioned in the Description section and if values are also correct (no unexpected values at least, if you want to calculate the standard deviation of the grades use [this](https://www.investopedia.com/terms/s/standarddeviation.asp))
4. Add a string to the grade scale names (instead of 5.0 for example) and make sure the standard grade deviation is not shown

### Screenshots


Table:
<img width="687" alt="Screenshot 2021-08-02 at 11 29 34" src="https://user-images.githubusercontent.com/44401560/127844186-8b9d47df-bda8-4c12-826a-5b5988437fc1.png">

Made graph bit thinner:
![Screenshot 2021-08-02 at 15 51 21](https://user-images.githubusercontent.com/44401560/127872638-01dbd46f-a4db-45c2-a82d-6532c2d4ef47.png)
Now:

<img width="814" alt="Screenshot 2021-08-28 at 13 35 53" src="https://user-images.githubusercontent.com/44401560/131216630-3afeb71c-002a-4225-adcf-27ad264dd5ad.png">


### Review Progress





- Code Review
- Manual Tests
